shared/templates/firefox_policy-setting/tests/missing_value.fail.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+firefox_cfg="policies.json"
+firefox_dirs="/usr/lib/firefox/distribution /usr/lib64/firefox/distribution /usr/local/lib/firefox/distribution /usr/local/lib64/firefox/distribution"
+{{{ find_python() }}}
+
+# Iterate over the possible Firefox install directories
+for firefox_dir in ${firefox_dirs}; do
+    # write our bad config in every location to ensure remediation fixes it.
+    mkdir -p ${firefox_dir}
+    echo "{ \"policies\" :{ } }" > ${firefox_dir}/${firefox_cfg}
```

Comment:
```suggestion
    echo '{ "policies" :{ } }' > "${firefox_dir}/${firefox_cfg}"
```

I propose to reduce escaping as much as possible.